components/notifications/index.tsx

```
@@ -18,26 +18,34 @@ const Notifications: React.FC<NotificationsProps> = ({
+      {
+        notifications.length === 0 ? (
+          <EmptyState
+            hideAction
+            image={require("@/assets/images/illustration2.png")}
+            subtitle="We have no notifications for you today!"
+            title="You are all caught up! "
+          />
+        ) : notifications.map((item) => (
+          <ScrollView
+            style={styles.container}
+          >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)